src/stories/LargeFilled.stories.tsx

```
@@ -0,0 +1,40 @@
+  args: {
+    children: '버튼 이름',
+    isActive: false,
+  },
+};
+
+export const Error: Story = {
+  args: {
+    children: '버튼 이름',
+    isError: true,
+  },
+};
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)